.gitlab-ci.yml

```diff
@@ -21,66 +33,88 @@ include:
     file: '.gitlab-ci-check-license.yml'
   - project: 'Northern.tech/Mender/mendertesting'
     file: '.gitlab-ci-github-status-updates.yml'
+  - project: 'Northern.tech/Mender/mendertesting'
+    file: '.gitlab-ci-check-docker-build.yml'
 
-test:unit:
-  stage: test
-  before_script:
-    - apt-get -qq update && apt-get install -yq $(cat deb-requirements.txt)
-    # start the dbus service
-    - service dbus start
+build:docker:
+  rules:
+    - when: never
+  script: ["true"]
```

Comment:
Isn't this optional with the `when: never` rule?

Reply:
Basically here we should be _overriding_ jobs from the template which have their `script`. If GitLab complained about script section being empty it means that the job was not added by the template in the first place, and hence no need for the `rules` to remove it.

You can use the [pipeline editor](https://gitlab.com/Northern.tech/northerntechhq/nt-connect/-/ci/editor?branch_name=main) while working on CI to quickly visualize and sanitize the pipeline.

Reply:
... but it would've been added if the branch was `staging`.